playbooks/container-runtime/private/config.yml

```diff
@@ -22,4 +22,4 @@
         name: container_runtime
         tasks_from: package_crio.yml
       when:
-        - openshift_use_crio | bool
+        - openshift_use_crio
```

Comment:
We definitely want to cast to bool here and anywhere else it's possible for a user to supply a boolean value because
openshift_use_crio=true is a string
and openshift_use_crio=True is a boolean.